src/scenic/simulators/carla/model.scenic

```diff
@@ -39,18 +39,36 @@ except ModuleNotFoundError:
                            'from this scenario')
 
 if 'carla_map' not in globalParameters:
-    raise RuntimeError('need to specify map before importing CARLA model '
-                       '(set the global parameter "carla_map")')
-simulator CarlaSimulator(globalParameters.carla_map)
-
-precipitation = Options({0: 70, 1: 30}) * Range(0, 100)
-param precipitation = precipitation
-param precipitation_deposits = Range(precipitation, 100)
-param cloudiness = Range(precipitation, 100)
-param wind_intensity = Range(0, 100)
-param sun_azimuth_angle = Range(0, 360)
-param sun_altitude_angle = Range(-90, 90)
-
+    param carla_map = None
+if 'address' not in globalParameters:
```

Comment:
You don't need any of these `if` statements. Just write `param carla_map = None`, etc.: the command-line option to set global parameters overrides all `param` statements.